### PR TITLE
Add support for Fitipower EK9716B LCD controller for CrowPanel 7.0" board

### PR DESCRIPTION
**Pull request by lboue**
## Add support for Fitipower EK9716B LCD controller

Add support for [Fitipower EK9716B](http://www.lcdwiki.com/res/MRG7111_7122/EK9716B_DS_rev1.5.pdf) LCD controller.

> EK9716 is a highly integrated 1200 channel source driver with TTL interface Timing Controller for color TFT-LCD panels. EK9716 integrated source driver, timing controller and pin control interface.
> EK9716 input timing support TTL digital 24bit parallel RGB data format, and source output support 8-bit resolution 256 gray scales with dithering features. Operating parameters can be set via pin control for all control features. Special circuit architecture is designed for lower power dissipation.
> EK9716 support two chip cascade operation mode to reduce the FPC amount and save the cost. Configure able Master and Slave configuration increase the flexibility for different panel design. With wide range of supply voltages and small output deviations make this chip more suitable for various applications.

## Tests

Tested with CrowPanel 7.0" board and [Squareline Simple Wi-Fi Clock Example](https://github.com/esp-arduino-libs/ESP32_Display_Panel/tree/master/examples/SquareLine/v8/WiFiClock).

IMG_1368

### CODE REVIEW

**Comment by Lzw655** [comment]
Hi @lboue, I noticed that you submitted several PRs (#75, #77, #79, #81) related to this issue. Could you consolidate all the changes into this PR? Once it's ready, I will review and merge it.

**Comment by lboue** [comment]
Ok I'll try to do that soon.

**Comment by lboue** [comment]
@Lzw655 Is this correct now?

File: docs/LCD_Controllers.md [hunk 1]
```diff
@@ -15,3 +15,4 @@
 | [ST7796](https://components.espressif.com/components/espressif/esp_lcd_st7796)   | 1.2.1       |
 | [ST77916](https://components.espressif.com/components/espressif/esp_lcd_st77916) | 0.0.2       |
 | [ST77922](https://components.espressif.com/components/espressif/esp_lcd_st77922) | 0.0.2       |
+| EK9716B                                                                          | 0.1.6       |
```
Comment (Lzw655) [inline docs/LCD_Controllers.md:18]:
```suggestion
| EK9716B                                                                          | -       |
```
If there's no version to fill, we can just use "-" instead.

**Review by Lzw655 — APPROVED** [review]
LGTM! Thank you for your contributions.

**Comment by lboue** [comment]
> LGTM! Thank you for your contributions.

I approved the suggestion.

**Pull request merged**